lib/elixir/lib/access.ex

```diff
@@ -1,23 +1,18 @@
 defmodule Access do
   @moduledoc """
-  Key-based access to data structures using the `data[key]` syntax.
```

Comment:
Yes, it was hard to fit in everything. Since Access is independent from the Elixir syntax, how about: "Provides generic key-based access to data structures". That sells it a little short, but I suppose there’s more room for detail in the later sections.